Assets/Src/Scripts/tests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Tests : MonoBehaviour
+{
```

Comment:
And, furthermore, this code is not following [the Google C# coding style](https://google.github.io/styleguide/csharp-style.html).